ccd-definition/CaseEvent/User/UserEventsLRspec.json

```diff
@@ -64,9 +64,9 @@
     "PreConditionState(s)": "AWAITING_RESPONDENT_ACKNOWLEDGEMENT",
     "PostConditionState": "*",
     "SecurityClassification": "Public",
-    "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/version/V_1/about-to-start",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/version/V_2/about-to-start",
```

Comment:
please create a prod file and a non prod file for this
1. move DEFENDANT_RESPONSE_SPEC and CLAIMANT_RESPONSE_SPEC old (master) code into prod file
2. move DEFENDANT_RESPONSE_SPEC and CLAIMANT_RESPONSE_SPEC new code into nonprod file
3. remove DEFENDANT_RESPONSE_SPEC and CLAIMANT_RESPONSE_SPEC from base file